### ISSUE TITLE
GIS.lab client - automatic swap partition detection is not working

### ISSUE THREAD

**Issue by imincik** [imincik/gis-lab#410]
It seems that automatic swap partition detection is not working and swap partitions are not mounted on client boot (tested on machines on FS CVUT). 

I use swap on my computer. Here is output from commands used for swap detection executed on my computer:

```
$ sudo find /dev -type b -regex ".*/[a-z]*"
/dev/sda

$ sudo gdisk -l /dev/sda
GPT fdisk (gdisk) version 0.8.8

Partition table scan:
  MBR: MBR only
  BSD: not present
  APM: not present
  GPT: not present


***************************************************************
Found invalid GPT and valid MBR; converting MBR to GPT format
in memory. 
***************************************************************

Disk /dev/sda: 1465149168 sectors, 698.6 GiB
Logical sector size: 512 bytes
Disk identifier (GUID): 88C54737-1F18-4968-ADDF-0A6507E73714
Partition table holds up to 128 entries
First usable sector is 34, last usable sector is 1465149134
Partitions will be aligned on 2048-sector boundaries
Total free space is 5805 sectors (2.8 MiB)

Number  Start (sector)    End (sector)  Size       Code  Name
   1            2048          499711   243.0 MiB   8300  Linux filesystem
   5          501760      1465147391   698.4 GiB   8300  Linux filesystem

```

So, no swap detected on my computer either.

**Comment by mpastor** [comment]
I can't see any swap partition. Can you send the content of '/proc/swaps' file from your running system (not GIS.lab client)?
```
$ cat /proc/swaps
```
Maybe you are using swap file instead of disk partition.

**Comment by imincik** [comment]
My machine:

$  cat /proc/swaps
Filename                                Type            Size    Used    Priority
/dev/dm-2                               partition       7966716 813704  -1

**Comment by imincik** [comment]
Seems that mkswap is missing in out test lab in FS CVUT

**Comment by imincik** [comment]
@mpastor , isn't it fixed in  2dcbe11b2d53dbefc222dafa9e07b48b0f07c599 ? If so, please close this issue.

**Comment by mpastor** [comment]
no, it isn't. it would be hard to fix this bug, because it's too complex to support all possible types of swap devices (mdadm, lvm...).

currently only swap that _EXISTS_ in running system is supported - that means, if you have swap partition on LVM, and the LVM is running, such swap will be used.
